internal/tool/ssh/impl/manager.go

```diff
@@ -87,11 +87,14 @@ type manager struct {
 	ctx        context.Context
 	dep        *protos.Deployment
 	logger     *slog.Logger
-	locations  []string // addresses of the locations
-	mgrAddress string   // manager address
+	mgrAddress string // manager address
 	registry   *status.Registry
 	started    time.Time
 
+	// keys: addresses of the locations
```

Comment:
managed locations, and the corresponding directories to store the weaver binaries at each location